Marlin/src/lcd/ultralcd.h

```diff
@@ -296,8 +296,8 @@ class MarlinUI {
       #if ENABLED(LCD_SET_PROGRESS_MANUALLY)
         static uint8_t progress_bar_percent;
         static void set_progress(const uint8_t progress) { progress_bar_percent = _MIN(progress, 100); }
-        static void set_progress_done() { set_progress(0x80 + 100); }
-        static void progress_reset() { if (progress_bar_percent & 0x80) set_progress(0); }
+        static void set_progress_done() { set_progress(100); }
```

Comment:
Although the estimation of completion may say "100" that might actually mean 99.5. The flag is used to clearly indicate completion and flags that the value should be cleared on the next interaction.